ci/repokitteh/modules/newpr.star

```diff
@@ -8,26 +8,52 @@ In the meantime, please take a look at the [contribution guidelines](https://git
 
 """
 
+DRAFT_MESSAGE = """
+As a reminder, PRs marked as draft will not be automatically assigned reviewers,
+or be handled by maintainer-oncall triage.
+
+Please mark your PR as ready when you want it to be reviewed!
+"""
+
+
 def get_pr_author_association(issue_number):
   return github.call(
     method="GET",
     path="repos/envoyproxy/envoy/pulls/%s" % issue_number)["json"]["author_association"]
```

Comment:
how does author_association different than author?

Reply:
not sure - this is existing repokitteh code, not from my change :-)